### ISSUE TITLE
generated results are terrible with bloom

### ISSUE THREAD

**Issue by raihan0824** [project-baize/baize-chatbot#30]
I use this repo to finetune bloomz-7b1-mt with alpaca data (50k conversation) and the results are terrible. It takes 8 hours to train with the same arguments as in how you finetune the llama. What could be the reason of this?

**Comment by guoday** [comment]
Firstly, alpaca data is not intended for conversation but rather for instructional purposes. You should consider leveraging other data specifically designed for conversation. It appears that Bloomz-7b1-mt is a model fine-tuned on other tasks, so it's hard to align using instruction data.

**Comment by raihan0824** [comment]
no, I did convert the data to the same format as in this repo. And when I use `bloomz-7b1-mt`on alpaca [repo](https://github.com/PhoebusSi/Alpaca-CoT), it works just fine, however, I want to make it conversational so I use this repo. I think the problem is the training hyperparameter because I find this repo's hyperparameter is different from this [repo](https://github.com/PhoebusSi/Alpaca-CoT). What do you think?

**Comment by raihan0824** [comment]
and for some reason, the `adapter_model.bin` file size is so small (<1MB) even though I trained for 8 hours. it's so weird

**Comment by guoday** [comment]
I see. Please adjust batch size or epochs [here](https://github.com/project-baize/baize-chatbot/blob/c02f40b664602f77d676e30d6f1b72a2f3ce205a/finetune.py#L22-L25), because you only update 50k/64= 781 steps. However, we update about 3000 steps using all data.
